Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -63,7 +62,7 @@ const CreateNewMaintenanceWindow = () => {
     repeat: 1,
     date: dayjs(),
     startTime: dayjs(),
-    duration: "60",
+    durationOptions: "60",
```

Comment:
This was fine, duration was a good name here.  I meant adjust the name of the `timeFrames` object to `durations` or `durationOptions` so it is clear what it is for